js/ui/context_menu/ui.menu_base.js

```diff
@@ -685,8 +685,10 @@ var MenuBase = HierarchicalCollectionWidget.inherit({
     },
 
     _itemClickHandler: function(e) {
+        if(e.cancel) return;
```

Comment:
Can we avoid a modification of event arguments?

Reply:
No, we can not use this flag on widget level because 2 different clicks should be handled but event propagation after 1 click should be prevented

Reply:
Can we use a more specific name for this? Use `_dx` prefix for example?